src/redesign/login/messages.jsx

```diff
@@ -119,10 +119,10 @@ const messages = defineMessages({
     defaultMessage: 'The email address you\'ve provided isn\'t formatted correctly.',
     description: 'Validation message that appears when email address format is incorrect',
   },
-  'email.format.validation.less.chars.message': {
-    id: 'email.format.validation.less.chars.message',
-    defaultMessage: 'Email must have at least 3 characters.',
-    description: 'Validation message that appears when email address is less than 3 characters',
+  'username.or.email.format.validation.less.chars.message': {
+    id: 'username.or.email.format.validation.less.chars.message',
+    defaultMessage: 'Username or Email must have at least 3 characters.',
```

Comment:
nit: the first letter for the email must be small.

Reply:
resolved 👍